### DISCUSSION TITLE
Error PickingImage using expo-image-picker

### DISCUSSION THREAD

**Discussion by bytesagar** [expo/expo#17097] [Archived: Questions]
Failure delivering result ResultInfo{who=null, request=2, result=-1, data=Intent { dat=content://com.android.providers.media.documents/document/image:2678 flg=0x1 }} to activity {com.globalpanacea.sif/com.globalpanacea.sif.MainActivity}: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter mImageLoader

**Comment by lukasz-szulborski** [comment]
Please follow the instructions from this answer https://github.com/expo/expo/issues/15119#issuecomment-1000268968

Long story short, run `expo install expo-image-loader` to fix this issue
